Bottom: f63888d0372e86697d078aa5425f992e2d0ed397
Top:    6ec00d68f38f5e62cfe23f7b602c114dd9e916af
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-20 15:56:37 +0100

Refresh of hid-gtk-some-noop-changes-to-t

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index be922a6..d7549ca 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1361,9 +1361,9 @@ SwapSides (int argc, char **argv, Coord x, Coord y)
   gint flipd;
   int do_flip_x = 0;
   int do_flip_y = 0;
+  int active_group = GetLayerGroupNumberByNumber (LayerStack[0]);
   int comp_group = GetLayerGroupNumberByNumber (component_silk_layer);
   int solder_group = GetLayerGroupNumberByNumber (solder_silk_layer);
-  int active_group = GetLayerGroupNumberByNumber (LayerStack[0]);
   bool comp_on = LAYER_PTR (PCB->LayerGroups.Entries[comp_group][0])->On;
   bool solder_on = LAYER_PTR (PCB->LayerGroups.Entries[solder_group][0])->On;
